### DISCUSSION TITLE
Can ASReview be used to find both pros and cons in a dataset?

### DISCUSSION THREAD

**Discussion by SagevdBrand** [asreview/asreview#567] [Q&A] [answered]
In the case that I would like to see pro's and cons of a specific strategy for example, is it possible for ASReview to find both of them? Or will it simply 'get stuck' on finding only the pro arguments of a strategy? 

If it does 'get stuck' are there any solutions to circumvent this?

**Comment by FelixWdm** [comment, marked as answer]
Assuming the dataset contains 'pros', 'cons' and 'neutral' or similar, you should include both at least a 'pro' and a 'con' as prior knowledge 'relevant' and start screening. Depending on the ratio of 'pro's' and 'cons' the risk of it getting stuck are not that high. You would simply regard both the ''pros' and 'cons' as relevant.
